tests/unit-tests/test_expression.cpp

```diff
@@ -226,6 +236,10 @@ namespace test
                     e3 = extract(v1, 8, 1),
                     e4 = concat(v2,v1),
                     e5 = ITE(v1, ITECond::LE, v2, v3, v4);
+            Constraint c1 = e1 == e2,
+                    c2 = e3 > e4,
+                    c3 = c1 && c3;
+            Expr    e6 = ITE(c3, v3, v4);
```

Comment:
By the way, the errors in the tests come from this. The expression sizes for `e1`, `e2` and `e3`,`e4` don't match so they can't be compared.

Reply:
Yeah saw that! Had another one where I accidentally referenced an uninitialized constraint. Tests pass locally now, the `CMakeUserPresets.json` setup is nice, should have just been doing that from the start

Reply:
Great! Also you didn't really need to create intermediate `Constraint` variables and could have written 
````

ITE(a==b, e1, e2)
````
instead of 
````
Constraint c = (a==b);
ITE(c, e1, e2)
````
But it doesn't really matter in the tests